lte/gateway/configs/gateway.mconfig

```diff
@@ -39,7 +39,9 @@
       "ipv6DnsAddress": "2001:4860:4860:0:0:0:0:8888",
       "natEnabled": true,
       "congestionControlEnabled": true,
-      "enableConvergedCore": false
+      "enableConvergedCore": false,
+      "defaultSliceServiceType": "1",
+      "defaultSliceDifferentiator" : "0x0"
```

Comment:
why is this default different than the defaults defined in other parts?

Reply:
Comments addressed. Changed it to 0xffffff